### PR TITLE
Prevent arrow conversion to emoji; fix alignment of ticket elements

### PR DESCRIPTION
**Pull request by grablair**
On Safari (on iPhone), the right arrow is being converted to a ➡️ emoji, which is not desirable (FWIW, the down arrow is fine 🤷🏻‍♂️).

This changes the arrows from unicode arrows to an Icon, which also animates as the tickets open / close.

Also, fixes the inverted boolean logic around `ticketIsCollapsed`. Same exact functionality, I just had the boolean logic backwards on all of it.

Also, better aligns all the elements in the ticket title button.

Screenshot 2024-10-31 at 13 18 56
Screenshot 2024-10-31 at 13 19 05

---

Also fixes some alignment of ticket elements to ensure that the ticket counter is always on the right. If wrapping is necessary, the name of the tier should wrap.

BEFORE:

22152AB2-87BE-4E63-8FB4-97A3D7983AFA_1_201_a

AFTER:

18A1FE2B-AFE8-4632-8B1A-E9CCAF13B4A4_4_5005_c

---

## Checklist

- [X] I have read the contributing guidelines.
- [X] My code is of good quality and follows the coding standards of the project.
- [X] I have tested my changes, and they work as expected.

Thank you for your contribution! 🎉

### CODE REVIEW

File: frontend/src/components/routes/ticket-widget/SelectTickets/index.tsx [hunk 1]
```diff
@@ -328,15 +328,14 @@ const SelectTickets = (props: SelectTicketsProps) => {
                                                 {(!ticket.is_available && ticket.type === 'TIERED') && (
                                                     <TicketAvailabilityMessage ticket={ticket} event={event}/>
                                                 )}
-
-                                                <span className={'hi-ticket-collapse-arrow'}>
-                                                    {ticketIsCollapsed ? '\u25BC' : '\u25B6'}
-                                                </span>
                                             </div>
+                                            <span className={`hi-ticket-collapse-arrow`}>
+                                                <IconNavigationFilled className={ticketIsCollapsed ? "" : "open"} />
```
Comment (daveearley) [inline frontend/src/components/routes/ticket-widget/SelectTickets/index.tsx:333]:
```suggestion
                                                <IconChevronRight className={ticketIsCollapsed ? "" : "open"} />
```

**Review by daveearley — COMMENTED** [review]
Thanks for this! I'd suggest using the [right chevron icon](https://tabler.io/icons/icon/chevron-right), as it's a more sleek and in keeping with other icons used in the app.

**Comment by grablair** [comment]
Ah yeah that'll be much better. I'll get that change in tomorrow (PDT).

**Comment by daveearley** [comment]
Looks great, thank you!

**Pull request merged**